lib/layer.js

```diff
@@ -28,23 +28,34 @@ var hasOwnProperty = Object.prototype.hasOwnProperty
 
 module.exports = Layer
 
-function Layer(path, options, fn) {
+function Layer(paths, options, fn) {
   if (!(this instanceof Layer)) {
-    return new Layer(path, options, fn)
+    return new Layer(paths, options, fn)
   }
 
-  debug('new %o', path)
+  debug('new %o', paths)
   var opts = options || {}
 
   this.handle = fn
   this.name = fn.name || '<anonymous>'
   this.params = undefined
   this.path = undefined
-  this.regexp = pathRegexp(path, this.keys = [], opts)
```

Comment:
The removal of `this.regexp` would be a semver major change, as there are many module out there that reach in for this property. If this is a required change here, can you make the target branch be `2.0` instead of `master`?

Reply:
I don't see an obvious way to make it backwards compatible, but I'm open to suggestions.

Otherwise, I can target 2.0. Just let me know what the consensus is.

Reply:
Updated to target 2.0